editor/editor.gd

```diff
@@ -57,7 +59,11 @@ func _ready() -> void:
 	print('a')
 
 func _process(_delta: float) -> void:
-	var prev_time := cur_time
+	var elapsed_os_time = 0.0
+	if root_viewport.music.playing:
+		elapsed_os_time = (OS.get_ticks_msec() * 0.001) - current_os_time
```

Comment:
What is the purpose of `elapsed_os_time` and `previous_music_time`? Shouldn't we be applying all calculations in this if statement?

Reply:
same as before, previous_music_time (renamed from prev_time) just helps determine whether to update the note positions or not so we don't run the update code while the music is stopped. elapsed_os_time is what I'm using to calculate the new current_music_time.